app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt

```diff
@@ -201,7 +201,10 @@ class AudioService : LifecycleService() {
                             .build()
                 }
             })
-
+            // override media connector increment speed
+            val incrementMs = resources.getInteger(R.integer.increment_ms)
+            setFastForwardIncrementMs(incrementMs)
+            setRewindIncrementMs(incrementMs)
             setPlayer(exoPlayer)
```

Comment:
Before add a new line before this line to keep each logical block of code separate

Reply:
Added new line